src/Jira/Api.php

```diff
@@ -106,6 +106,9 @@ public function setEndPoint($url)
     {
         $this->fields = array();
 
+        //Regular expression to remove trailing slash
```

Comment:
Please change the following:

1. add a space after `//`
2. add a dot at the end of the comment text